queries.sql

```diff
@@ -46,3 +47,28 @@ SELECT neutered, MAX(escaped_attempts) as excaped_attempts FROM animals GROUP BY
 SELECT species, MIN(weight_kg) as minWeight, MAX(weight_kg) as maxWeight FROM animals GROUP BY species
 SELECT species, AVG(escaped_attempts) as avgEscapes FROM animals WHERE date_part('year', date_of_birth) 
 BETWEEN 1990 AND 2000 GROUP BY species
+
+/* DAY THREE */
+
+SELECT name as animal_name FROM animals a JOIN owners o on a.owner_id = o.id 
+	WHERE o.id IN (SELECT id FROM owners WHERE full_name = 'Melody Pond');
+
+SELECT a.name FROM animals a JOIN species s on a.species_id = s.id 
+	WHERE s.id IN (SELECT id FROM species WHERE s.name = 'Pokemon');
+
+SELECT o.full_name as owner, a.name as animal FROM animals a RIGHT JOIN owners o ON a.owner_id = o.id
+
+SELECT count(*) as total, s.name as species 
+  FROM animals a JOIN species s ON a.species_id = s.id GROUP BY s.name
+	
+SELECT a.* FROM animals a JOIN species s ON a.species_id = s.id Join owners o ON a.owner_id = o.id WHERE
+	s.id IN (SELECT id FROM species WHERE s.name = 'Digimon') AND 
+	o.id IN (SELECT id FROM owners WHERE o.full_name = 'Jennifer Orwell');
+	
+SELECT a.* FROM animals a Join owners o ON a.owner_id = o.id WHERE
+	o.id IN (SELECT id FROM owners WHERE o.full_name = 'Dean Winchester') AND 
+	a.escaped_attempts = 0;
+
+SELECT count(*) as total, o.full_name as OwnerName FROM 
+	animals a JOIN owners o ON a.owner_id = o.id 
```

Comment:
- [ ] Here we have to find `Who owns the most animals?`
- Here should be `JOIN owners o ON a.species_id = o.id`